sparse/core.py

```diff
@@ -1380,3 +1380,24 @@ def _match_arrays(a, b):
     b_idx[bic[1:-1]] -= bsw[bi[:-1]] + bl[bi[:-1]] - 1
     b_idx = np.cumsum(b_idx)
     return a_idx, b_idx
+
+
+def random(shape, density=0.01, dtype=None):
+    """ Generate a random sparse multidimensional array
+
+    Parameters
+    ----------
+    shape: :obj:`tuple` of :obj:`int`
+        Shape of the array
+    density: :obj:`float`
+        Density of the generated array.
+    dtype : dtype
+        Type of the returned array values.
+
+
+    """
+    elements = np.prod(shape)
+
+    return COO.from_scipy_sparse(
+        scipy.sparse.rand(elements, 1, density, dtype=dtype)
```

Comment:
Here, it would be useful to specify `format='coo'` explicitly. Might speed up the conversion.

Reply:
`scipy.sparse.rand()` defaults to `format='coo'`...